### DISCUSSION TITLE
Squest Operator

### DISCUSSION THREAD

**Discussion by theneedyguy** [HewlettPackard/squest#770] [Show and tell]
I would like to share my Squest Operator. It is still in an early phase but can already deploy a fully functioning Squest instance. It's to help users create and manage a Squest instance on a Kubernetes cluster with ease. [Link to the Repo](https://github.com/theneedyguy/squest-operator)

**Comment by Sispheor** [comment]
Hi,
Thanks for sharing! We re always happy to see community project around Squest 🙂.
It definitively something we have in our pipe but unfortunately no time to start working on it. The Ansible deployment was a first step.
We'll take a look to what you did and see what can be integrated soon or later in the core.
